fluffy/templates/home.html

```diff
@@ -35,14 +35,21 @@
         <form class="pastebinForm" method="POST" action="/paste">
             <p class="switch-modes"><a>upload files instead</a></p>
 
-            <p>
-                <textarea id="text" name="text" placeholder="paste text here" required autofocus>{{text}}</textarea>
+            <p id="pastedText">
+                <textarea id="text" name="text" placeholder="paste text here" autofocus>{{text}}</textarea>
             </p>
+
+            <div style="display: none" id="pastedTwoText">
+                <textarea id="diff" name="diff" placeholder="paste text here for document 1" autofocus style="width:49%; flex:1; margin: 2px;">{{text}}</textarea>
+                <textarea id="diff2" name="diff2" placeholder="paste text here for document 2" autofocus style="width:49%; flex:1; margin: 2px;">{{text2}}</textarea>
```

Comment:
How does mixing the widths and margins like this work across screen sizes? I would think the remaining 2% is a different size across different screen widths.

Is it better to use the `column-gap` CSS property?